staff/serena-mejias/cv/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <section>
```

Comment:
I miss a header in your cv!

Reply:
also the main content would be nice to envelop it in `<main>` tag? take a look at https://www.w3schools.com/tags/tag_main.asp

Reply:
Done.

Reply:
Done.